nengo_viz/components/netgraph.py

```diff
@@ -27,15 +27,21 @@ def __init__(self, viz, config, uid):
         self.parents = {}
         self.networks_to_search = [self.viz.model]
         self.initialized_pan_and_zoom = False
-        self.last_modify_time = os.path.getmtime(self.viz.viz.filename)
+        try:
+            self.last_modify_time = os.path.getmtime(self.viz.viz.filename)
+        except:
+            self.last_modify_time = None;
         self.last_reload_check = time.time()
 
     def check_for_reload(self):
-        t = os.path.getmtime(self.viz.viz.filename)
+        try:
+            t = os.path.getmtime(self.viz.viz.filename)
 
-        if self.last_modify_time < t:
-            self.reload()
-            self.last_modify_time = t
+            if self.last_modify_time < t:
+                self.reload()
+                self.last_modify_time = t
+        except:
+            pass
```

Comment:
With the filename not existing (because there is no physical file of the model for the IPython notebook). But also in the regular case, it would be better if the server continues running if the file is deleted.